lib/tools.js

```diff
@@ -1168,7 +1165,7 @@ function getCertificateInfo(cert) {
  *
  * @alias generateDefaultCertificates
  * @memberof Tools
- * @returns {defaultPrivate: any[],defaultPublic: any[]}
+ * @returns {{ defaultPrivate: any[];defaultPublic: any[] }}
```

Comment:
To declare an object return type in JSDoc, you need two pairs of braces. One for the comment, one for the type